website/pages/docs/integrate/nia-integration.mdx

```diff
@@ -42,17 +42,19 @@ Consul-Terraform-Sync compatible Terraform module development process is fairly
 
 - Consul [documentation](https://www.consul.io/docs)
 - Consul-Terraform-Sync use case [documentation](https://www.consul.io/use-cases/network-infrastructure-automation)
-- Writing Consul-Terraform-Sync compatible Terraform modules [guide](Will link to NIA Tech Preview section - Requirements - Network Integration - How to Create a Module)
+- Writing Consul-Terraform-Sync compatible Terraform modules [guide](https://www.consul.io/docs/nia/installation/requirements#how-to-create-a-terraform-module)
```

Comment:
Ah sorry about this @BeenzSyed! I just changed the section title to "How to Create a Compatible Terraform Module" so the anchor is now `#how-to-create-a-compatible-terraform-module#how-to-create-a-compatible-terraform-module`

([docs preview](https://deploy-preview-8887--consul-docs-preview.netlify.app/docs/nia/installation/requirements#how-to-create-a-compatible-terraform-module))

Reply:
👍 No problem! Thank you - I updated the links with the correct anchor.